### DISCUSSION TITLE
Support for arm linux

### DISCUSSION THREAD

**Discussion by brianluby** [wavetermdev/waveterm#580] [Ideas]
Do you have plans to support ARM processors for linux? I run UTM/Qemu on M1 mac and would like to be able to run wave on linux vm.

**Comment by esimkowitz** [comment]
Hi @brianluby, we don't currently have any runners capable of compiling for Linux ARM. We are signed up to get the GitHub Actions ARM runners when they become available, at which time we plan to support this.

**Reply by yjlc-pc** [reply]
Hi! @esimkowitz Are you have a arm64 phone or pad?
if you have,I can tell you how to use the phont or pad build your program.

**Reply by esimkowitz** [reply]
We have access to the GitHub ARM Linux runners now so I am planning on getting the arm64 support addressed soon.

**Comment by brianluby** [comment]
Excellent. Look forward to testing it.
